configuration/pih/petl/jobs/data_kind/encounter/job.yml

```diff
@@ -0,0 +1,10 @@
+type: "sqlserver-bulk-import"
+configuration:
+  extract:
+    datasource: "mysql/openmrs.yml"
+    query:  "data_kind/encounter/source.sql"
+
+  load:
+    datasource: "sqlserver/openmrs_extractions.yml"
+    table: "datakind_encounter"
+    schema: "data_kind/encounter/target.sql"
```

Comment:
I'm wondering if we should think about adding a condition here to only run on JJ Dossen and Mirebalais?  

I created a ticket about adding support for this: https://pihemr.atlassian.net/browse/PETL-100